doc/src/quickstart/index.rst

```diff
@@ -332,14 +305,14 @@ However, there is an issue with the above BLIF produced by ABC: the latches (ris
 
 Re-inserting clocks
 ^^^^^^^^^^^^^^^^^^^
-We will restore the clock information by running a script which will transfer that information from the original ODIN BLIF file (writing it to the new file ``blink.pre-vpr.blif``):
+We will restore the clock information by running a script which will transfer that information from the original Parmys BLIF file (writing it to the new file ``blink.pre-vpr.blif``):
 
 .. code-block:: bash
 
     > $VTR_ROOT/vtr_flow/scripts/restore_multiclock_latch.pl \
-        blink.odin.blif \
-        blink.abc_no_clock.blif \
-        blink.pre-vpr.blif
+        ~/vtr_work/quickstart/blink_manual/temp/blink.parmys.blif \
```

Comment:
Does run_vtr_flow directly call abc and the restore_multiclock_latch.pl script when run with parmys?  If not, I think we should leave ABC and this script out of the parmys section, and just refer to them in the Odin II section (which does need them).